mountinfo/mounted_linux_test.go

```diff
@@ -314,6 +316,47 @@ func TestMountedBy(t *testing.T) {
 				}
 			}
 
+			// Check the public MountedFast() function as a whole.
+			mounted, sure, err := MountedFast(m)
```

Comment:
Also see my other comment about this test becoming quite complex; wondering if we should just move this to a separate (`TestMountedFast`) test. There will be _some_ boilerplating / code duplication, but perhaps it's a good trade-off compared to the complexity we're ending up into now. i.e. using `subtests` normally avoids having to include the function name in the `t.Errorf`, but because we're now combining so many we have to do it again.

Reply:
Yes, this looks better. Updated.